apps/dotcom-worker/src/routes/extractBookmarkMetadata.tsx

```diff
@@ -0,0 +1,30 @@
+import { assert } from '@tldraw/utils'
+import { handleExtractBookmarkMetadataRequest } from '@tldraw/worker-shared'
+import { IRequest } from 'itty-router'
+import { Environment } from '../types'
+
+export async function extractBookmarkMetadata(request: IRequest, env: Environment) {
+	if (request.method === 'GET') {
+		// legacy route: extract metadata without saving image
+		return handleExtractBookmarkMetadataRequest({ request })
+	}
```

Comment:
why not just keep only the GET route and do the uploading there?

Reply:
discussed offline